lib/services/computeManagement2/package.json

```diff
@@ -12,7 +12,7 @@
     "Zavery, Amar <amzavery@microsoft.com>",
     "Sun, Carissa <caris@microsoft.com>"
   ],
-  "version": "1.0.0-preview",
+  "version": "2.0.0-preview",
```

Comment:
do you require actually require a major version update? If the changes are not breaking and are incremental, you could just increment the minor version. That way, people who use semantic versioning `^1.0.0` would also get your update.